src/util/transifex-upload.js

```diff
@@ -0,0 +1,79 @@
+const apiBaseUrl = 'https://www.transifex.com/api/2/project';
+const resource = 'app-strings';
+const token = '1/9492ad42eff9cc8c2d5a141f8df3647570d7a641';
```

Comment:
Should this be in some env?

Reply:
yes and now we're going to have to update it everywhere :angry:

Reply:
it looks like this one may be just for this project, so possibly not as bad